file-formats/nont/type/zif_aff_nont_v1.intf.abap

```diff
@@ -0,0 +1,35 @@
+INTERFACE zif_aff_nont_v1
+  PUBLIC.
+
+  TYPES:
+
+    BEGIN OF ty_main,
+      "! $required
+      format_version  TYPE zif_aff_types_v1=>ty_format_version,
+
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header          TYPE zif_aff_types_v1=>ty_header_60_cloud,
+
+      "! <p class="shorttext">Name</p>
+      "! Node type name in camel case
+      "! $required
+      name            TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Expanded Name</p>
+      "! Expanded name
+      "! $required
```

Comment:
Here https://github.com/SAP/abap-file-formats/pull/408#issue-1423538776
you described this field. Do you want to add a better description to the abap doc of this field?

Reply:
Thanks for the feedback! Yes, I have already started to improve the abap doc comments. I will commit these changes next week in one go.